tools/gsc-mannwhitney_de/mannwhitney_de.xml

```diff
@@ -0,0 +1,103 @@
+        <param name="factor2" type="text" label="The second value that the comparison factor can take." help="typical values could be 'LOW', 'wt', 't2', etc."/>
+        <param name="fdr" type="float" value="0.01" label="FDR threshold"
+               help="Reject tests if adjusted p-values (Benjamini-Hochberg correction) is superior to the FDR thresold."/>
+    </inputs>
+    <outputs>
+        <data name="output" format="tabular" label="Results of Mann-Whitney differential analysis from ${on_string}" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input" value="filterCells_100.tsv" ftype="txt"/>
+            <param name="sep" value='tab' />
+            <param name="colnames" value="TRUE"/>
+            <param name="comparison_factor_file" value="signature_2columns.tsv" ftype="tabular"/>
+            <param name="factor1" value="HIGH"/>
+            <param name="factor2" value="LOW"/>
+            <param name="fdr" value="0.01"/>
+            <output name="output" file="geneMetadata.tab" ftype="tabular"/>
+        </test>
+    </tests>
+    <help>
+
+**What it does**
+
+The tools takes a table of gene expression values (e.E. log10(CPM+1), etc...) from single cell RNAseq sequencing libraries (columns) and a metadata file that contains at least two columns :
+    * Cell identifiers
+    * Column that differentiate cell in two groups. It must be a column with only two categories (factor levels).
+
+For each gene (rows in expression data file), this script perform a Mann-Whitney test between 
+the two groups of cells (high/low, mutant/wild type) and then adjust the returned p-values by using the
+Benjamini-Hochberg (BH) correction. A False Discovery Rate (FDR) threshold is used to determine if gene
+can be considerated as significant (p-adjust below FDR) or not (p-adjust above the FDR value). 
+
+**Output**
+
+The tools returns a result table. For each row (genes) :
+
+==================== ======================================================================================================
+              Column Description
+-------------------- ------------------------------------------------------------------------------------------------------
+                mean mean expression across all cells
+                  SD standard deviation of its expression across all cells
+            variance variance of expression across all cells
+Percentage_detection (number of cells where the gene is detected (expression value superior to 0) / number of cells) * 100
+        mean_factor2 mean expression across cells of the second group
+        mean_factor1 mean expression across cells of the first group
+         fold_change mean_factor1 - mean_factor2
+           statistic W statistic of Mann-Whitney test
+             p.value p-value of Mann-Whitney test
+            p.adjust p-values adjusted from the BH correction
+         Significant TRUE/FALSE vector (if p-adjust inferior to FDR then the value is TRUE) 
```

Comment:
p-adjust et FDR sont équivalents dans leur définition 'le FDR étant une p-adjust en utilisant la BH correction'
Je remplacerais donc "FDR" par "FDR cut-off"

Reply:
ok pour moi

Reply:
Déjà, je remplacerais BH par Benjamini etc... 
Ensuite @NairaNaouar pourquoi quotes tu `le FDR étant une p-adjust en utilisant la BH correction` ? Je ne comprends pas cette expression. Tu vois bien que si je remplace ton quote dans l'explication de @bellenger-l , ça donne : (if p-adjust inferior to `p-adjust en utilisant la BH correction` then the value is TRUE), ce qui n'est pas logique du tout.

On a déjà parlé de cela avec Léa: il y a une ambiguïté car les gens utilisent souvent "FDR" à tort et à travers. C'est l'occasion d'être clair !

Reply:
Je pense que ce que @NairaNaouar voulait dire c'est dans ma phrase mettre : `if p-adjust inferior to the FDR cut-off then the value is TRUE` mais j'ai peut être mal compris

Reply:
@bellenger-l oui c'est exactement ça

Reply:
Donc je suis entrain de faire vos dernières suggestions, j'ai intégré celle-ci